test/chemistry/test_driver_methods.py

```diff
@@ -39,6 +42,7 @@ def _run_driver(driver, transformation=TransformationType.FULL,
                     freeze_core=True):
         qmolecule = driver.run()
 
+        warnings.filterwarnings('ignore', category=DeprecationWarning)
```

Comment:
This test is around drivers not the core Hamiltonian that is just used to aid the test. We can simply replace this by the FermionicTransformation there is not need to generate a new set of tests where the names get added with gsc since gsc is not the prime goal of these tests rather its to test the drivers,